src/viam/sdk/registry/registry.hpp

```diff
@@ -22,9 +22,9 @@
 namespace viam {
 namespace sdk {
 
-// TODO(RSDK-1742): instead of std::functions, consider making these functions
+// TODO(RSDK-3030): instead of std::functions, consider making these functions
```

Comment:
Changed some of these TODOs as that original ticket is now closed and we should be thinking about these changes as part of [RSDK-3030](https://viam.atlassian.net/browse/RSDK-3030).